dwave/cloud/cli.py

```diff
@@ -314,26 +297,34 @@ def _config_create(config_file, profile, ask_full=False, auto_token=False):
             click.echo(f"Available profiles: {', '.join(existing)}")
         default_profile = next(iter(existing))
 
-        _note = " (select existing or create new)" if config_file_exists else ""
-        profile = default_text_input(f"Profile{_note}", default_profile)
+        if ask_full:
+            _note = " (select existing or create new)" if config_file_exists else ""
```

Comment:
Maybe I can't see it, but does this get used anywhere? If no then it can probably be dropped. If yes, I think you need to specify a `_note` in the `else` block

Reply:
You mean `_note` variable? It's used in the line below. And in case config file doesn't exist, note should be empty.

Reply:
So it is! I just can't read. Ignore me